circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -1144,6 +1144,39 @@ class CircularSeekBar @JvmOverloads constructor(
         recalculateAll()
     }
 
+    private fun customPaint(paint: Paint, block: (Paint) -> Unit) {
```

Comment:
To show a more complete meaning for these methods, do you think adding a verb prefix for the naming would make more sense? Such as "applyCustomXXX".